scripts/spec.sh

```diff
@@ -38,14 +38,14 @@ PEREGRINE_PLAIN=$TMP_DIR"peregrine-kilt.plain.spec"
 PEREGRINE_JQ=$TMP_DIR"peregrine-kilt.json"
 PEREGRINE_OUTPUT=dev-specs/kilt-parachain/peregrine-kilt.json
 
-docker run $RELAY_CHAIN_IMG build-spec --chain rococo-local --disable-default-bootnode > $RELAY_PEREGRINE_PLAIN
-$TMP_DIR/kilt-parachain build-spec --runtime spiritnet --chain spiritnet-dev --disable-default-bootnode > $PEREGRINE_PLAIN
```

Comment:
Did you intend to remove the space before `$PEREGRINE_PLAIN`?

Reply:
that's the formatter... 🙄